src/main/java/com/epam/test_generator/services/exceptions/ProjectClosedException.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.test_generator.services.exceptions;
+
+public class ProjectClosedException extends RuntimeException {
```

Comment:
Provide javadoc on how to use this exception

Reply:
When to throw and when to catch. And when caught, how to exctract additional data from it i.e. name of project that have already been closed.

Reply:
fixed